samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserMemStorage.spec.ts

```diff
@@ -99,19 +99,6 @@ describe("In Memory Storage Tests", function () {
             await page.close();
         });
 
-        it("Performs loginRedirect", async () => {
```

Comment:
This e2e test has been removed as loginRedirect is not supported with memory storage now that `storeAuthStateInCookies` is removed.